exporters/otlp/otlp.go

```diff
@@ -231,3 +239,51 @@ func (e *Exporter) uploadTraces(ctx context.Context, sdl []*tracesdk.SpanData) e
 	}
 	return err
 }
+
+func NewExportPipeline(ctx context.Context, exportOpts []ExporterOption,
+	resOpts []resource.Option) (*Exporter, *sdktrace.TracerProvider, *push.Controller, error) {
+
+	exp, err := NewExporter(ctx, exportOpts...)
+	if err != nil {
+		return nil, nil, nil, err
+	}
+
+	res, err := resource.New(ctx, resOpts...)
+	if err != nil {
+		return nil, nil, nil, err
+	}
+
+	bsp := sdktrace.NewBatchSpanProcessor(exp)
+	tracerProvider := sdktrace.NewTracerProvider(
+		sdktrace.WithConfig(sdktrace.Config{DefaultSampler: sdktrace.AlwaysSample()}),
+		sdktrace.WithResource(res),
+		sdktrace.WithSpanProcessor(bsp),
+	)
+
+	pusher := push.New(
+		basic.New(
+			simple.NewWithExactDistribution(),
+			exp,
+		),
+		exp,
+		push.WithPeriod(7*time.Second),
+	)
+
+	return exp, tracerProvider, pusher, nil
+}
+
+func InstallNewPipeline(ctx context.Context, exportOpts []ExporterOption,
+	resOpts []resource.Option) (*Exporter, *sdktrace.TracerProvider, *push.Controller, error) {
+
+	exp, tp, pusher, err := NewExportPipeline(ctx, exportOpts, resOpts)
+	if err != nil {
+		return nil, nil, nil, err
+	}
+
+	otel.SetTextMapPropagator(propagation.TraceContext{})
```

Comment:
I don't think any other installers set the propagators.  Why do you want to do it here?

Reply:
For the same reason as i add the resource https://github.com/open-telemetry/opentelemetry-collector/pull/2159 but removing it will make more consistent with others installers it's probably better to remove it.

Reply:
I'd suggest a similar thing as stated in https://github.com/open-telemetry/opentelemetry-go/issues/1391: rather than modify all the exporter installers with a propagators option, use the global default (which should be set correctly).